package.json

```diff
@@ -4,41 +4,40 @@
   "description": "",
   "author": "",
   "devDependencies": {
+    "babel-core": "^6.18.2",
```

Comment:
babel-core is already a dependency of gulp-babel. https://github.com/babel/gulp-babel/blob/master/package.json#L35
So unless they move it out to a peer dependency we would like to stick with their version